libpod/define/pod_inspect.go

```diff
@@ -51,6 +53,10 @@ type InspectPodData struct {
 	// Containers gives a brief summary of all containers in the pod and
 	// their current status.
 	Containers []InspectPodContainerInfo `json:"Containers,omitempty"`
+	// CPUs contains the total number of executable CPUs in the pod
+	CPUs float64 `json:"cpus,omitempty"`
+	// ResourceLimits contains linux specific CPU data for the pod
+	ResourceLimits *spec.LinuxResources `json:"resource_limits,omitempty"`
```

Comment:
I wouldn't throw this entire struct in Inspect - we should output fields from it, but the struct itself is not intended to be human-readable